### ISSUE TITLE
Gracefully remove the container when Ctrl-C

### ISSUE THREAD

**Issue by eiso** [src-d/gitbase-playground#210]
When I run gitbase via docker, and Control-C the server, it gracefully exits and removes the container from docker engine. When I do the same on the playground, I still have to manually remove it with `docker rm -f gitbase-playground`, I don't think there is any need to persist the container, I would remove it like we do with gitbase or bblfsh.

**Comment by bzz** [comment]
I belive using `docker-compose down` after `ctrl-c` is the solution for this issue.

If that works, it shall be documented in https://github.com/src-d/gitbase-playground#with-docker-compose

**Comment by eiso** [comment]
I meant without docker-compose. I don't know how gitbase does it, but it cleans up the exited container.

**Comment by bzz** [comment]
@eiso if you want a single container to be deleted, you need to add `--rm`  [option to the run command](https://docs.docker.com/engine/reference/run/#clean-up---rm).

So it shall be 
```
docker run -it --rm \
    --name gitbase-playground \
    --network host \
    --env GITBASEPG_DB_CONNECTION="root@tcp(127.0.0.1:3306)/none" \
    --env GITBASEPG_SELECT_LIMIT=0 \
    srcd/gitbase-playground:latest
```

**Comment by eiso** [comment]
Of course! [insert facepalm gif] [insert embarrassed gif]

**Comment by bzz** [comment]
N.p!

**Comment by bzz** [comment]
Closing as issue seems to be resolved and #213 is merged
